scipy/linalg/__init__.py

```diff
@@ -5,6 +5,15 @@
 
 .. currentmodule:: scipy.linalg
 
+.. toctree::
+   :hidden:
+
+   linalg.blas
+   linalg.cython_blas
+   linalg.cython_lapack
+   linalg.interpolative
+   linalg.lapack
```

Comment:
As with `scipy.optimize.cython_optimize`, these are now only mentioned in "See Also" sections. Maybe they should appear in a top-level listing on the `linalg` page? It looks like the choice to make the `toctree` hidden was intentional. Fine with me either way.